.github/workflows/publish_imgs.yml

```diff
@@ -0,0 +1,155 @@
+# 5. Once we logged in, we then execute the script "publish-imgs", which will publish the image passed into it.
+
+name: Publish Docker Images
+
+on:
+  # This job will be fired off when a comment has been made in an issue (which also impliitly includes PRs)
+  issue_comment:
+    types: [created]
+
+jobs:
+  build:
+    name: Publish Image to Dockerhub
+    
+    # Running on a GitHub hosted runner
+    runs-on: ubuntu-18.04
+
+    # Checks if the comment contains the word 'publish'
+    # As well as if the comment was made in a PR
+    if: > 
+      startsWith(github.event.comment.body, 'publish')
+      && startsWith(github.event.issue.pull_request.url, 'https://')
+
+    steps:
+      # Checks out this repository to the runner
+      - uses: actions/checkout@v2
+
+      # Installs pre-reqs the runner does not initially have
+      # TODO: This can be cached in the future
+      - name: Install Pre-Reqs
+        id: prereq
+        run: |
+          set -x
+          curl -o install-nix-2.3.6 https://releases.nixos.org/nix/nix-2.3.6/install
+          curl -o install-nix-2.3.6.asc https://releases.nixos.org/nix/nix-2.3.6/install.asc
+          gpg2 --recv-keys B541D55301270E0BCF15CA5D8170B4726D7198DE
+          gpg2 --verify ./install-nix-2.3.6.asc
+          sh ./install-nix-2.3.6
+
+          . /etc/os-release
+          sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/x${NAME}_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
+          wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/x${NAME}_${VERSION_ID}/Release.key -O- | sudo apt-key add -
+          sudo apt-get update -qq
+          sudo apt-get install skopeo
+        continue-on-error: false
+
+      # Verifies if the commenter is an approver, otherwise, the job will output an error
+      - name: Verify that Commenter is Approver
+        id: verify
+        run: |
+          resp=$(curl -sSf \
+          --url https://api.github.com/orgs/Nebulaworks/teams/approvers/memberships/${{ github.event.comment.actor.login }} \
+          --header 'Authorization: Bearer ${{ secrets.NWIAUTO_PAT }}' \
+          --header 'Content-Type: application/json' | jq -r '.state' \
+          )
+
+          if [ $resp == "active" ]; then
+            echo ::set-output name=status::success
+          else
+            echo ::set-output name=status::failure
+          fi
+        continue-on-error: true
+
+      # If the commenter is an approver, attempts to login to docker.io as nwiauto
+      - name: Login To Docker Hub as nwiauto
+        id: login
+        env:
+          USER: nwiauto
+        if: steps.verify.outputs.status == 'success'
+        run: |
+          echo "${{ secrets.NWIAUTO_PASSWORD }}" | docker login docker.io --username "$USER" --password-stdin
```

Comment:
Ah didnt realize that with the GitHub runner and docker. The password be `encoded` at base64 not something thats encrypted although it might appear that way from an initial glance.

I would just say for simplicity we should just leverage the `echo` and `heredoc`. That way we arent relying on any docker components that were implied from the base vm.

Reply:
Right, my mistake haha. I agree, in my testing, I just confirmed the heredoc function works, so I'll be sticking with that function for logging in.